boilerplates/express/files/express-entry.ts

```diff
@@ -14,6 +18,24 @@ const __dirname = dirname(__filename);
 const isProduction = process.env.NODE_ENV === "production";
 const root = __dirname;
 
+/*{ @if (it.BATI.has("firebase-auth")) }*/
+let admin: App | undefined;
+
+function initializeAdminApp() {
+  return initializeApp({
+    credential: applicationDefault(),
+  });
+}
+
+if (!getApps().length) {
+  admin = initializeAdminApp();
+} else {
+  admin = getApp();
+  deleteApp(admin);
+  admin = initializeAdminApp();
+}
```

Comment:
@phonzammi What are you trying to do here? Especially the `else` part

Reply:
It's just to make sure to initialize a singleton firebase app